extensions/wikia/Forum/RelatedForumDiscussionController.class.php

```diff
@@ -39,8 +39,8 @@ public static function purgeCache( $threadId ) {
 			$wikiaPage = WikiPage::newFromID( $id );
 			if ( $wikiaPage ) {
 				$wikiaPage->doPurge();
+				self::logError( "Found a null related wikipage on thread purge", [ "articleId" => $id, "threadId" => $threadId ] );
```

Comment:
I believe this shouldn't have been moved from the `else` below?

Reply:
We might want to consider removing it outright as I don't think anybody is monitoring this error anyways.